metadata_xml/cioos_template.jinja2

```diff
@@ -245,6 +244,74 @@
   </mdb:dateInfo>
   {% endif %}
 
+  {# metadataStandard: Recommended #}
+  <mdb:metadataStandard>
+    <cit:CI_Citation>
+      <cit:title>
+        <gco:CharacterString>ISO 19115-1 Geographic information - Metadata</gco:CharacterString>
+      </cit:title>
+      <cit:edition>
+        <gco:CharacterString>First Edition 2014-04-01</gco:CharacterString>
+      </cit:edition>
+    </cit:CI_Citation>
+  </mdb:metadataStandard>
+  <mdb:metadataStandard>
+    <cit:CI_Citation>
+      <cit:title>
+      <gco:CharacterString>ISO 19115-2 Geographic information - Metadata</gco:CharacterString>
+      </cit:title>
+      <cit:edition>
+        <gco:CharacterString>Second Edition 2019-01</gco:CharacterString>
+      </cit:edition>
+    </cit:CI_Citation>
+  </mdb:metadataStandard>
+
+  {# metadataProfile: Optional, will be CIOOS core Mandatory once profile is published #}
+	<mdb:metadataProfile>
+    <cit:CI_Citation>
+      <cit:title>
+        <gco:CharacterString>Canadian Integrated Ocean Observing System Metadata Profile of ISO 19115</gco:CharacterString>
+      </cit:title>
+
+     <cit:date>
+        {# date: mandatory #}
+       <cit:CI_Date>
+            <cit:date><gco:Date>2019-11-06</gco:Date></cit:date>
+        {# dateType: mandatory #}
+        <cit:dateType>
+        {# CIOOS core mandatory element #}
+        {# MI_Metadata/dateInfo/CI_Date/date/CI_DateTypeCode #}
+          <cit:CI_DateTypeCode codeList="http://standards.iso.org/iso/19115/resources/Codelists/cat/codelists.xml#CI_DateTypeCode" codeListValue="publication">publication</cit:CI_DateTypeCode>
+        </cit:dateType>
+        </cit:CI_Date>
+      </cit:date>
+    </cit:CI_Citation>
+  </mdb:metadataProfile>
```

Comment:
Nice to see this section in here now! Minor point but it looks like the indentation might be a bit off here.

Reply:
The indentation is off a bit throughout, I am thinking once we get through this PR I can submit another one only to indent the entire document